### DISCUSSION TITLE
%essentials_home_<number>_<world>% placeholder

### DISCUSSION THREAD

**Discussion by Rawxdealz** [EssentialsX/Essentials#4899] [Ideas and Feature Suggestions] [answered]
%essentials_home_<number>_<x|y|z>% these placeholders exist, but not the world one. Is it possible to create that? And is it possible to let the %essentials_home_<number>_<x|y|z>% placeholder not return 10 numbers, but like -315,5 or 12.5

**Comment by Rawxdealz** [comment]
![image](https://user-images.githubusercontent.com/42896942/164946931-f9d6b716-fdaa-45d2-af3b-c6618058e677.png)

**Comment by Rawxdealz** [comment]
![image](https://user-images.githubusercontent.com/42896942/164947122-7b43cd8c-8f62-4038-b48c-2441b7960968.png)

**Comment by osfanbuff63** [comment, marked as answer]
According to @mdcfe in #4859:

> We don't develop the PlaceholderAPI Essentials expansion. You can request this feature at the [PAPI expansion repository](https://github.com/placeholderapi/essentials-expansion).

**Reply by Rawxdealz** [reply]
Thanks for the response. I will do that.
